staging/src/k8s.io/client-go/tools/cache/delta_fifo.go

```diff
@@ -161,12 +161,14 @@ type DeltaFIFO struct {
 	lock sync.RWMutex
 	cond sync.Cond
 
+	// We maintain the properties that:
+	// - a key is in `items` if and only if that key is in `queue`;
+	// - a given key appears in at most one position in `queue`;
+	// - each Deltas in `items` has at least one Delta.
```

Comment:
What's your thoughts of moving this block below the definition of `queue` to minimize forward references?

Reply:
Reorganized comments to remove forward references.